doc/release-notes.md

```diff
@@ -19,44 +20,239 @@ dashd/dash-qt (on Linux).
+- SPV clients can now detect and track Dash-specific transactions
+- Feature parity between bloom filters and compact block filters
+- Protection against serving incorrect filter data to light clients
 
-Documentation Updates
+Other notable changes
 ---------------------
 
-- Updated translation documentation with current Transifex links and fixed typos to help contributors properly access translation resources (dash#6739).
+* To help prevent fingerprinting transactions created by the Dash Core wallet, change output
+  amounts are now randomized. (#6685)
+* Dash Core will no longer migrate EvoDb databases generated in v19 and v20, users upgrading
+  from these versions are recommended to run `-reindex` to rebuild all databases and indexes. (#6579)
+
+Updated REST APIs
+-----------------
+
+- The `/headers/` and `/blockfilterheaders/` endpoints have been updated to use
+  a query parameter instead of a path parameter to specify the result count. The
+  count parameter is now optional and defaults to 5 for both endpoints. The old
+  endpoints are still functional, and have no documented behaviour change. (#24098)
+
+  For `/headers`, use
+  `GET /rest/headers/<BLOCK-HASH>.<bin|hex|json>?count=<COUNT=5>` instead of
+  `GET /rest/headers/<COUNT>/<BLOCK-HASH>.<bin|hex|json>` (deprecated)
+
+  For `/blockfilterheaders/`, use
+  `GET /rest/blockfilterheaders/<FILTERTYPE>/<BLOCK-HASH>.<bin|hex|json>?count=<COUNT=5>` instead of
+  `GET /rest/blockfilterheaders/<FILTERTYPE>/<COUNT>/<BLOCK-HASH>.<bin|hex|json>` (deprecated)
+
+P2P and network changes
+-----------------------
+
+- `MIN_PEER_PROTO_VERSION` has been bumped to `70221`. (#6877)
+- `cycleHash` field in `isdlock` message will now represent a DKG cycle starting block of the signing quorum instead of a DKG cycle starting block corresponding to the current chain height. While this is fully backwards compatible with older versions of Dash Core, other implementations might not be expecting this, so the P2P protocol version was bumped to 70237. (#6608)
+- UNIX domain sockets can now be used for proxy connections. Set `-onion` or `-proxy` to the local socket path with the prefix `unix:` (e.g. `-onion=unix:/home/me/torsocket`). UNIX socket paths are now accepted for `-zmqpubrawblock` and `-zmqpubrawtx` with the format `-zmqpubrawtx=unix:/path/to/file`. (#6634)
+
+Updated RPCs
+------------
+
+* The keys `platformP2PPort` and `platformHTTPPort` have been deprecated for the following RPCs, `decoderawtransaction`,
+  `decodepsbt`, `getblock`, `getrawtransaction`, `gettransaction`, `masternode status` (only the `dmnState` key),
+  `protx diff`, `protx listdiff` and has been replaced with the key `addresses`.
+  * The deprecated key is still available without additional runtime arguments, but is liable to be removed in future versions
+    of Dash Core. (#6811)
+
+* `protx revoke` will now use the legacy scheme version for legacy masternodes instead of defaulting to the
```

Comment:
protx's related changes are very massive

I would suggest to group all protx / address related changes together, or at least post them close to each other (now they are separated by `instantsentoaddress` and `bls generate`